swap/src/main.rs

```diff
@@ -13,47 +13,43 @@
 #![forbid(unsafe_code)]
 
 use anyhow::{bail, Result};
+use cli::Options;
+use config::Config;
 use futures::{channel::mpsc, StreamExt};
 use libp2p::Multiaddr;
 use log::LevelFilter;
-use std::{io, io::Write, process};
+use std::{io, io::Write, path::Path, process};
 use structopt::StructOpt;
+use swap::{alice, bitcoin::Wallet, bob, config, Cmd, Rsp, SwapAmounts};
 use tracing::info;
-use url::Url;
+use xmr_btc::bitcoin::{BroadcastSignedTransaction, BuildTxLockPsbt, SignTxLock};
 
 mod cli;
 mod trace;
 
-use cli::Options;
-use swap::{alice, bitcoin::Wallet, bob, Cmd, Rsp, SwapAmounts};
-use xmr_btc::bitcoin::{BroadcastSignedTransaction, BuildTxLockPsbt, SignTxLock};
-
 // TODO: Add root seed file instead of generating new seed each run.
 // TODO: Remove all instances of the todo! macro
 
-// TODO: Add a config file with these in it.
-// Alice's address and port until we have a config file.
-pub const PORT: u16 = 9876; // Arbitrarily chosen.
-pub const ADDR: &str = "127.0.0.1";
-pub const BITCOIND_JSON_RPC_URL: &str = "127.0.0.1:8332";
-
 #[tokio::main]
 async fn main() -> Result<()> {
-    let opt = Options::from_args();
+    let opt: Options = Options::from_args();
```

Comment:
Do we really need this change?

Reply:
nope, my IDE was unable to resolve the type. I can remove it.